docs/pipeline_configuration.md

```diff
@@ -0,0 +1,75 @@
+## Overview
+Creating pipelines in Conduit is a simple process with many steps to define the resources and their configurations, this
+could be done through API or Conduit's UI. But to make this process even simpler and to make it possible to load 
+pre-defined pipelines, we introduce **_Pipeline Configurations_**.
```

Comment:
IMO we should add the word "file" somewhere in the mix (e.g. pipeline configuration files). I would also add it in the name of the file itself (e.g. `pipeline_configuration_files.md`).